WordPressKit/WordPressOrgXMLRPCApi.swift

```diff
@@ -26,6 +28,14 @@ open class WordPressOrgXMLRPCApi: NSObject {
     ///
     @objc public static let minimumSupportedVersion = "4.0"
 
+    private lazy var urlSession: URLSession = makeSession(configuration: .default)
+    private lazy var uploadURLSession: URLSession = {
+        if self.backgroundUploads {
+            return makeSession(configuration: .background(withIdentifier: self.backgroundSessionIdentifier))
```

Comment:
Nitpick: I tried locally, and it compiled without the explicit `self.`

```suggestion
        if backgroundUploads {
            return makeSession(configuration: .background(withIdentifier: backgroundSessionIdentifier))
```